.circleci/config.yml

```diff
@@ -85,14 +80,13 @@ install_dependencies: &install_dependencies
       name: Install dependencies
       command: |
         docker exec -it pthd pip install -r requirements-dev.txt
-        export install_ignite_cmd='pip install .'
+        export install_ignite_cmd='python setup.py install'
```

Comment:
The change is here, the rest should be formatting by pre-commit
I have checked this is the only change, if you find other else where, let me know.

Reply:
Yes, I also think this is the only change.